website/docusaurus.config.js

```diff
@@ -41,7 +41,25 @@ module.exports = {
         { to: 'https://github.com/h2oai/wave/discussions', label: 'Discuss', position: 'left' },
         { to: 'https://h2oai.github.io/h2o-ai-cloud/', label: 'Enterprise', position: 'left', },
         { to: 'https://github.com/h2oai/wave/releases', label: 'Download', position: 'right', },
-        { href: 'https://github.com/h2oai/wave', label: 'GitHub', position: 'right', },
+        { href: 'https://github.com/h2oai/wave',
+          position: 'right',
+          className: 'navbar-icon navbar-icon-github',
+          'aria-label': 'GitHub repository',
+          html: `<img src="img/github.svg"/>`,},
```

Comment:
We should use absolute URLs instead of relative ones. Notice how the icons break when you navigate to `Get started` page for example.

Also please add an `alt` attribute to the icons so that they are accessible.